text/0064-publishing-protobufs.md

```diff
@@ -0,0 +1,198 @@
+binary data representation.
+
+The current Protobuf definitions for communicating with fog and consensus are
+defined in the main 
+MobileCoin Foundation git [repo](https://github.com/mobilecoinfoundation/mobilecoin). 
+
+Most clients end up submoduling the MobileCoin Foundation repo in order to
+generate code at build time.  The MobileCoin Foundation repo is fairly
+large and submoduling it to gain access to a few files is not very ergonomic.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Protobuf definition files are made available in the following ways:
+
+1. [Buf Schema Registry](#buf-schema-registry)
+2. [Rust crates](#rust-crates)
+3. [The Protobuf git repo](#the-protobuf-git-repo)
+
+## Buf Schema Registry
+
+The [Buf Schema Registry](https://buf.build/explore/) is an online hosting
+platform of Protobuf defintions. It provides documentation hosting and a command
+line tool to detect breaking changes in Protobuf definitions.
+
+The MobileCoin Foundation Protobuf files are available at
+<https://buf.build/mobilecoinfoundation>.
+
+
+For most clients, the Buf Schema Registry is the recommended way to use the
+MobileCoin Foundation's Protobufs.
+
+## Rust Crates
+
+The MobileCoin Foundation Protobuf defintions are available via
+[Rust](https://www.rust-lang.org/) crates.  The crates use
+[prost](https://docs.rs/prost/latest/prost/) generated messages. Both
+[tonic](https://docs.rs/tonic/latest/tonic/) and
+[grpcio](https://docs.rs/grpcio/latest/grpcio/) are supported for the services.
+
+
+These crates can be found on
+<https://crates.io> by searching for one of the following:
+
+- [mc-*-grpcio](https://crates.io/search?q=mc-*-grpcio) for rust
+  [grpcio](https://docs.rs/grpcio/latest/grpcio/) services.
+- [mc-*-messages](https://crates.io/search?q=mc-*-messages) for
+  [prost](https://docs.rs/prost/latest/prost/) messages.
+- [mc-*-tonic](https://crates.io/search?q=mc-*-tonic) for
+  [tonic](https://docs.rs/tonic/latest/tonic/) services.
+
+Building the crates requires the client to have the Protocol Buffer Compiler,
+[protoc](https://grpc.io/docs/protoc-installation/), installed.
+
+For rust projects, the rust crates are the recommended way of using the MobileCoin Foundatin Protobuf defintions.
+
+## The Protobuf Git Repo
+
+Clients can directly access the Protobuf definitions via the git repo,
+<https://github.com/mobilecoinfoundation/protobufs>. 
+
+The other two methods should be prefered over direct access to the git repo, but
+there are times when it is necessary to use the repo directly.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The MobileCoin Foundation Protobuf defintions are located at
+<https://github.com/mobilecoinfoundation/protobufs>. When a series of git
+changes are deemed ready to release, the new versions will be published to
+<https://buf.build/mobilecoinfoundation> and crates published to <https://crates.io>. 
```

Comment:
Note buf.build name. I think this was the name we had settled on.  There is https://buf.build/mobilecoin but I recall we wanted "foundation" in the protobufs.